runtime/nls/j9cl/j9jcl_fr.nls

```diff
@@ -444,147 +444,165 @@ J9NLS_JCL_DUPLICATE_CLASS_DEFINITION.system_action=The JVM will throw a LinkageE
 J9NLS_JCL_DUPLICATE_CLASS_DEFINITION.user_response=Ensure there is no duplicate definition for the class in the message and try again.
 # END NON-TRANSLATABLE
 
-J9NLS_JCL_CRIU_FAILED_TO_OPEN_DIR=Impossible d'ouvrir le r\u00e9pertoire de donn\u00e9es du point de contr\u00f4le, errno=%li
+J9NLS_JCL_CRIU_FAILED_TO_OPEN_DIR=
 # START NON-TRANSLATABLE
-J9NLS_JCL_CRIU_FAILED_TO_OPEN_DIR.sample_input_1=1
-J9NLS_JCL_CRIU_FAILED_TO_OPEN_DIR.explanation=An error occurred when the JVM attempted to open the specified directory.
-J9NLS_JCL_CRIU_FAILED_TO_OPEN_DIR.system_action=The JVM will throw a JVMCheckpointException.
-J9NLS_JCL_CRIU_FAILED_TO_OPEN_DIR.user_response=Ensure the specified directory is accessible.
+J9NLS_JCL_CRIU_FAILED_TO_OPEN_DIR.sample_input_1=
```

Comment:
Perhaps we should have an issue to improve the tool: it recognizes that there's no sample input, but it could simply remove this line instead.

Reply:
It's not a fault of the tool, the English file still contains the line. I can remove these unused sample in another PR and the next round of translation will remove them from the translated files. I don't really want to redo this one for something that doesn't matter.

Reply:
I hadn't thought of it that way. I agree the right way to fix that is to remove the useless lines from the English file.
And you're right, it shouldn't have any effect on the resulting JDK.